### PR TITLE
New package: obs-v4l2sink-3.5

### PR DESCRIPTION
**Pull request by xslendix**
I found this plugin for OBS _extremly_ useful. I thought "might as well add this to void's packages". I am sure there's a better way to implement this but this is what I could have done for my first xbps package. If anyone can improve it, that would be awesome!

### CODE REVIEW

File: srcpkgs/obs-v4l2sink/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+archs="x86_64"
+depends="obs v4l2loopback"
+short_desc="OBS plugin that provides output capabilities to a v4L2 device"
+maintainer="xSlendiX <radu.ungureanu@techie.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/CatxFish/obs-v4l2sink"
+
+do_install() {
+	mkdir -r /usr/lib/obs-plugins
+	mkdir -r /usr/share/obs/obs-plugins/v4l2sink/locale
```
Comment (ericonr) [inline srcpkgs/obs-v4l2sink/template:14]: Why did you create this folder? Nothing's being installed into it.

File: srcpkgs/obs-v4l2sink/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'obs-v4l2sink'
+pkgname=obs-v4l2sink
+version=3.5
+revision=1
+archs="x86_64"
```
Comment (ericonr) [inline srcpkgs/obs-v4l2sink/template:5]: No need to restrict archs if it works on others.

File: srcpkgs/obs-v4l2sink/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+depends="obs v4l2loopback"
+short_desc="OBS plugin that provides output capabilities to a v4L2 device"
+maintainer="xSlendiX <radu.ungureanu@techie.com>"
+license="GPL-3.0-or-later"
```
Comment (ericonr) [inline srcpkgs/obs-v4l2sink/template:9]: It's `GPL-2.0 -or-later` https://github.com/CatxFish/obs-v4l2sink/blob/master/src/v4l2sink.cpp

File: srcpkgs/obs-v4l2sink/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+homepage="https://github.com/CatxFish/obs-v4l2sink"
+
+do_install() {
+	mkdir -r /usr/lib/obs-plugins
+	mkdir -r /usr/share/obs/obs-plugins/v4l2sink/locale
+
+	vinstall v4l2sink.so 755 /usr/lib/obs-plugins
+	vinstall *.ini 644 /usr/lib/obs-plugins
```
Comment (ericonr) [inline srcpkgs/obs-v4l2sink/template:17]: Install paths shouldn't start with `/`. Installing multiple files should be done with a `for file in *.ini` loop.

File: srcpkgs/obs-v4l2sink/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+archs="x86_64"
+depends="obs v4l2loopback"
+short_desc="OBS plugin that provides output capabilities to a v4L2 device"
+maintainer="xSlendiX <radu.ungureanu@techie.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/CatxFish/obs-v4l2sink"
+
+do_install() {
+	mkdir -r /usr/lib/obs-plugins
```
Comment (ericonr) [inline srcpkgs/obs-v4l2sink/template:13]: doesn't need to be created if you're using `vinstall` to put things in it.

**Review by ericonr — COMMENTED** [review]
Please test building templates before making PRs. This is missing distfiles and checksum fields, and no build steps are being run at all.

File: srcpkgs/obs-v4l2sink/template (hunk 4 above)
Comment (xslendix) [reply srcpkgs/obs-v4l2sink/template]: for file in *.ini doesnt work. it takes * literally

**Comment by HadetTheUndying** [comment]
OBS Includes this functionality by default in the latest release. There's no need for additional plugins.

**Comment by ericonr** [comment]
For the record, putting a binary file in `files/` isn't acceptable....

Thanks @HadetTheUndying, will close for now.

**Pull request closed without merging**